test/policies/ownership_policy_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+
+class OwnershipPolicyTest < ActiveSupport::TestCase
+  setup do
+    @authorizer = FactoryBot.create(:user)
+    @rubygem = FactoryBot.create(:rubygem, owners: [@authorizer])
+    @confirmed_ownership = @rubygem.ownerships.first
+    @unconfirmed_ownership = FactoryBot.build(:ownership, :unconfirmed, rubygem: @rubygem, authorizer: @authorizer)
+
+    @invited = @unconfirmed_ownership.user
+    @user = FactoryBot.create(:user)
+  end
+
+  def test_scope
+    @unconfirmed_ownership.save!
```

Comment:
Can we replace the `build` with `create` to remove this line? Otherwise leave a note for why this is needed.

Reply:
Yeah, the logic was that a "create" should be tested on an unsaved one, but it's somewhat arbitrary. I removed the scope tests.